Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNext(_ viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    var stackView: UIStackView!
+    var subtitleLabel: UILabel!
+    
+    private var titleLabel: UILabel!
+    private var imageView: UIImageView!
+    private var nextButton: UIButton!
+
+    private var onboardingTitle: String!
+    private var onboardingSubtitle: String!
+    private var onboardingImage: UIImage?
```

Comment:
These three properties can be `private let`s since they are being passed in the initializer